### ISSUE TITLE
[ActionButton]: quiet/selected state now has lower contrast with quick actions

### ISSUE THREAD

**Issue by adixon-adobe** [adobe/spectrum-css#1106]
## Description
Action Button's quiet / selected state is too subtle in a quick action bar. 

## Steps to reproduce

You can reproduce this pretty easily by adding the `is-selected` class to one of the quick action examples here: https://opensource.adobe.com/spectrum-css/quickaction.html

## Expected behavior
There should be more contrast between the background and the selected state. In a previous version the selected state was Spectrum Gray 300 (not 200). There was also a border which was a bug.

## Screenshots
<img width="1084" alt="Screen Shot 2021-01-19 at 11 34 58 AM" src="https://user-images.githubusercontent.com/48841507/105071545-5da36080-5a4a-11eb-9635-d00936075d35.png">

**Comment by castastrophe** [comment]
Quick actions have been deprecated so I'm going to close this. If we find it's relevant to another pattern, we can always re-open or create a fresh one for it!
